regression/pages/studio/course_outline_page.py

```diff
@@ -0,0 +1,72 @@
+"""
+Course Outline Page for Studio
+"""
+
+from edxapp_acceptance.pages.studio.overview import CourseOutlinePage
+
+from regression.pages.studio.utils import \
+    click_confirmation_prompt_primary_button
+from regression.tests.helpers import get_url
+
+
+class CourseOutlinePageExtended(CourseOutlinePage):
+    """
+    Coure Outline Extended Page for Studio
+    """
+    @property
+    def url(self):
+        """
+        Construct a URL to the page within the course.
+        """
+        return get_url(self.url_path, self.course_info)
+
+    def add_section_with_name(self, text):
+        """
+        Adds Section clicking the (main) New Section button and given name
+        """
```

Comment:
@raeeschachar Can you update the docstring as per [edx python guide](http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python_guidelines.html#docstrings)